subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -503,7 +503,7 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 				PR_WARNING("Passphrase provided without security configuration\n");
 			}
 			break;
-		case 'c':
+		case 'c': {
```

Comment:
Could simply add ; to force empty statement
`case 'c': ;`
or declare channel outside the case statement

Reply:
I think the comment you mentioned in the issue is good option then, having `;` itself you cause some confusion.